testem.js

```diff
@@ -8,12 +8,12 @@ module.exports = {
   launch_in_dev: [
     'Chrome'
   ],
-  'browser_args': {
-      'Chrome': [
-        '--disable-gpu',
-        '--headless',
-        '--remote-debugging-port=9222',
-        '--window-size=1440,900'
-      ],
-  }
+  // 'browser_args': {
```

Comment:
@trentmwillis helping us out yet again: https://twitter.com/trentmwillis/status/878277832278802432

Basically, we can make it headless only on CI...